oma-contents/src/searcher.rs

```diff
@@ -87,6 +87,36 @@ impl Mode {
     }
 }
 
+/// Automatically selects a search method based on whether or not the `rg` binary is used.
+///
+/// If the `rg` binary exists, the `rg` search is called, and if it doesn't, the fallback
```

Comment:
If the `rg` binary exists, oma calls `rg` for string search, otherwise, an internal string search method is used.